spec/module.dd

```diff
@@ -127,9 +127,9 @@ module c.stdio; // module stdio in the c package
 ---------
 
 $(P By convention, package and module names are all lower case. This is because
-those names can have a one-to-one correspondence with the operating system's
-directory and file names, and many file systems are not case sensitive. All
-lower case package and module names will minimize problems moving projects
+these names can have a one-to-one correspondence with the operating system's
```

Comment:
I would drop the "can". It doesn't work in practice to have a different module name than path.